README.md

```diff
@@ -60,7 +60,7 @@ dependencies {
 
 How To (Compiling Jar From Source)
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, we recommend use of JDK 11, maven, and an internet connection.
```

Comment:
```suggestion
To compile Paper, you need JDK 8 (we recommend at least JDK 11), maven, and an internet connection. 
```

Keeps the grammar consistent and modifies the correct words.

Reply:
I don't want to see jdk 8 in there at all. especially since it will be a hard requirement in a few months.

Reply:
Should we just say "You need Java 11" then? Circles back around, we're having conflicting arguments here. Current requirements are 8, future is 11, there's no better way to word it.

Reply:
Yeah... forget about suggest, xd